tests/test_sphinx_autodoc_typehints.py

```diff
@@ -115,14 +115,16 @@ def test_process_docstring_slot_wrapper():
     assert not lines
 
 
+@pytest.mark.parametrize('always_document_param_types', [True, False])
 @pytest.mark.sphinx('text', testroot='dummy')
-def test_sphinx_output(app, status, warning):
+def test_sphinx_output(app, request, status, warning, always_document_param_types):
```

Comment:
Why add the request fixture when you're not using it?

Reply:
Yeah, this was a leftover from what I tried earlier